### DISCUSSION TITLE
Obsidian Demo

### DISCUSSION THREAD

**Discussion by EmanuelFaria** [petermr/pyami#21] [General]
Manual and software creation of an Obsidian graph for IPCC reports

**Comment by EmanuelFaria** [comment]
<img width="2771" alt="Screen Shot 2022-11-25 at 9 57 11 AM" src="https://user-images.githubusercontent.com/9612595/204010862-82a65d9d-22b4-4da1-8ef3-0ad74a246887.png">

<img width="2793" alt="Screen Shot 2022-11-25 at 10 00 37 AM" src="https://user-images.githubusercontent.com/9612595/204011440-ea347768-82fb-4ac9-b178-a81c3fb58726.png">

**Comment by petermr** [comment]
# obsdian demo from joel chan

https://github.com/joelchan/semantic-climate-dg

## index.md
```
[[IPCC_AR6_WGIII_Chapter_02]]
- [[Providing access to modern energy services universally would increase global GHG emissions by at most a few percent]]
	- [[The additional energy demand needed to support decent living standards for all is estimated to be well below current average energy consumption]]
	- [[More equitable income distributions can reduce carbon emissions, but the nature of this relationship can vary by level of income and development]]
- [[It is possible to coordinate internationally to substantially reduce emissions of particular substances]]
	- [[CFC levels dropped precipitously over the course of 10 years in response to coordinated policies to ban the substance]]```


![Screenshot 2022-11-25 at 15 13 09](https://user-images.githubusercontent.com/733515/204013965-2112efed-78aa-44da-a28a-1a13c28ed5dd.png)

**Comment by EmanuelFaria** [comment]
URL to example of embedding pop-up windows in our enriched html

https://www.reddit.com/r/ObsidianMD/comments/vc7w6y/can_you_hover_the_mouse_over_an_external_link_and/

TIP: To find the link in this page that shows the demonstration and the code for javascript, css, html, etc, look for this phrase in the text:

Here's an example of that happening on an HTML page via JavaScript and an event handler.
([link](http://jsfiddle.net/yboss/q29tP/)
Or just start here: http://jsfiddle.net/yboss/q29tP
